circe.el

```diff
@@ -1489,6 +1500,19 @@ PATTERNS should be the list of regular expressions."
           (throw 'return t)))
       nil)))
 
+(defun circe-tracking-get-face (buffer)
+  "Return face for a given buffer. If the buffer has already some face
+specified, use it. This covers channels where our name was mentioned. Otherwise
+decide whether the buffer is for a channel, query or else and use a face based
+on this."
+  (with-current-buffer buffer
+    (cond ((get-text-property 0 'face buffer))
```

Comment:
It seems redundant to specify `buffer` when `buffer` is current. Especially if the `tracking-get-face` call exploits that fact.